configuration/pih/pih-config-haiti-cerca.json

```diff
@@ -27,7 +24,8 @@
     "todaysVisits",
     "visitManagement",
     "visitNote",
-    "uhmVitals"
+    "uhmVitals",
+    "labResults"
```

Comment:
Nit, but can we move this up so it is alphabetized?  it makes it easier to do a comparison between the config files for two different sites.

Reply:
Thanks this is alphabetized.